common/static/common/js/discussion/discussion.js

```diff
@@ -36,7 +36,7 @@
                 this.pages = options.pages || 1;
                 this.current_page = 1;
                 this.sort_preference = options.sort;
-                this.is_commentable_cohorted = options.is_commentable_cohorted;
+                this.is_commentable_divided = options.is_commentable_divided;
```

Comment:
nit: JS style standard is camelCase.  You might have issues with jslint. I have also noticed that underscore separators seem rampant in this file, so the change is up to you.